### ISSUE TITLE
Touchscreen support

### ISSUE THREAD

**Issue by evgen-y2k** [libredeb/lightpad#31]
I tested the application on my tablet computer and found that the application just closes if I try to scroll pages same like I do it in GNOME shell. All other, like click (touchscreen tap) on icon or points in bottom side of launcher works perfect. Also text input using virtual onscreen keyboard working fully correct. Could you please add support of touchscreen for pages scrolling?

**Comment by libredeb** [comment]
Yes, we need to move from GTK+ 3.0 to LibHandy... but i need to learn LibHandy first. Be patient!

**Comment by tio-trom** [comment]
I am following this issue. I wonder if there is any work o this still or anything planned? We would love to make a layout for touchscreens for our TROMjaro Layout Switcher https://www.tromjaro.com/ - we can test it a lot!
